DOCS-10985-dw-multipart-sample-kt (#77)
mulesoft/docs-dataweave: modules/ROOT/pages/dataweave-formats-multipart.adoc

------myboundary------
----

[[example4]]
=== Example: Iterate through Multipart payload

> This example shows how DataWeave iterates through through all the parts of the payload and process the message individually.

Reviewer: ```suggestion
This example shows how DataWeave iterates through all the parts of the payload and processes the each part individually.
```

Reviewer: "and processes the message individually" is unclear. I assume it means "each part of the message"? I'm not sure but that's what I suggested above.

=== Source
[source,dataweave,linenums]
----
%dw 2.0
output application/json
var multi = '--34b21
Content-Disposition: form-data; name="text"
Content-Type: text/plain

Book
--34b21
Content-Disposition: form-data; name="file1"; filename="a.json"
Content-Type: application/json

> {{{}}

Reviewer 2: ```suggestion
{
```

"title":"Java 8 in Action",
"author":"Mario Fusco",
"year":2014
}
--34b21
Content-Disposition: form-data; name="file2"; filename="a.html"
Content-Type: text/html

<!DOCTYPE html>
<title>
Available for download!
</title>

> -34b21-'

Reviewer 2: ```suggestion
--34b21--'
```
